jquery.floatThead.js

```diff
@@ -420,8 +420,8 @@
           }
           $table.css(layoutFixed);
           $floatTable.css(layoutFixed);
+          $header.after($newHeader);
```

Comment:
Ill need to find out the exact reason why this change is required before i merge this